docs/concepts/sides.md

```diff
@@ -58,3 +58,16 @@ Whenever you want to send information from one logical side to another, you must
 This is actually very commonly inadvertently done through static fields. Since the logical client and logical server share the same JVM in a single player scenario, both threads writing to and reading from static fields will cause all sorts of race conditions and the classical issues associated with threading.
 
 This mistake can also be made explicitly by accessing physical client-only classes such as `Minecraft` from common code that runs or can run on the logical server. This mistake is easy to miss for beginners, who debug in a physical client. The code will work there, but will immediately crash on a physical server.
+
+
+Writing Side-Only Mods
```

Comment:
Perhaps "one sided" would be a better term here?

Reply:
Yeah, that sounds better. Will use that.